src/main/webapp/script.js

```diff
@@ -314,4 +314,30 @@ function createUserLiElement(user) {
   buttonElement.innerText = user.name;
   liElement.appendChild(buttonElement);
   return liElement;
+}
+
+/** Creates an element that represents a task, including its delete button.
+  * @param {Hashmap} user
+  * @return {Element} HTML
+  */
+function createUserElement(user) {
```

Comment:
what will createUserElement() be used for vs. createUserLiElement()?

Reply:
Not sure. Wrote the second one before checking to see if one was already written. My bad.